frontend/server/src/main/java/org/pytorch/serve/util/ConfigManager.java

```diff
@@ -544,6 +546,14 @@ private int getIntProperty(String key, int def) {
         return Integer.parseInt(value);
     }
 
+    public int getDefaultBatchSize() {
```

Comment:
I think we will need doc updates here?

Reply:
we should mention about these defaults in docs

Reply:
Please see my comment on the corresponding issue 

https://github.com/pytorch/serve/issues/64#issuecomment-622292784

Reply:
I also added my thoughts on #64